weave/trace/op_type.py

```diff
@@ -224,7 +251,7 @@ def get_code_deps(
         ]
         return {"import_code": [], "code": [], "warnings": warnings}
 
-    source = textwrap.dedent(inspect.getsource(fn))
+    source = get_source_or_fallback(fn)
```

Comment:
This is not just called on ops, but if it hits the fallback, it will make functions called "op_fn".

So this whole algorithm can generate a bunch of code like this

```
def op_fn(*args, **kwargs):  # type: ignore
    # Code-capture unavailable for this op
    pass

def op_fn(*args, **kwargs):  # type: ignore
    # Code-capture unavailable for this op
    pass
    
...
```

Where we repeat the same op_fn block over and over again. 

I think we need to name these "blank functions" what the function we're trying to save would have been called.

You should write more tests to show this behavior.